cmd/notation/blob/policy/import.go

```diff
@@ -34,10 +34,10 @@ func importCmd() *cobra.Command {
 	var opts importOpts
 	command := &cobra.Command{
 		Use:   "import [flags] <file_path>",
-		Short: "Import blob trust policy configuration from a JSON file",
-		Long: `Import blob trust policy configuration from a JSON file.
+		Short: "Import blob trust policy file from a JSON file",
+		Long: `Import blob trust policy file from a JSON file.
 
-Example - Import blob trust policy configuration from a file:
+Example - Import blob trust policy file from a file:
```

Comment:
Can we make this sentence more meaningful? Something like:
```suggestion
Example - Import blob trust policy configuration from a file and store as `trustpolicy.blob.json`:
```

Reply:
Updated.